### ISSUE TITLE
Default branch

### ISSUE THREAD

**Issue by dmpetrov** [iterative/dvc-checkpoints-mnist#11]
Is there any specific reason for using `live` branch as default? My first thought after cloning it - something is broken 😅

**Comment by dberenbaum** [comment]
Yes, I've been struggling with this and looking for feedback on it! The reason that `live` is default now is because I  considered it the default workflow for most users. However, I think it's easy to argue that `make_checkpoint` might be a more reasonable default workflow for users since it doesn't have the `dvclive` dependency, or that `basic` should be the default branch since it's the simplest starting point.

**Comment by iesahin** [comment]
My 2c:

I think having a `main` branch with only the README and documenting the differences for each branch may be better.
